ractive-koa/app.js

```diff
@@ -51,8 +55,11 @@ ss.client.templateEngine.use(require('ss-ractive'), '/', {
 // - - - - - - - - - - - - - - - - - - - - - - - - >>>>> ROUTING & KOA
 
 app.use(bodyParser());
-app.keys = [config.get('session:secret')];
-app.use(session(app));
+app.use(require('koa-static')(__dirname + '/client/static'));
+app.use(function*(next) {
+	yield ssMiddleware.bind(null, this.req, this.res);
```

Comment:
Here's the binding. Technically, Koa has it's own set of middleware. Koa doesn't have passed variables `req` or `res`, so we're sort of "reverse binding." This bind happens on every Koa request. This is probably inefficient, since now we're running two sets of middleware—Koa's and SocketStreams. But it works!

I'd like to see at some future date the option to run SocketStream against generators, thus a more true concatenation of middleware could happen.

As it stands, we can now access the session via `this.req.session` in Koa. (Koa's native request is also still available as `this.request`: two separate request objects—like I said, less efficient.)